tests/integration/kubernetes/kata-deploy-test-usig-nfd.bats

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	node_name="$(kubectl get node -o name)"
```

Comment:
You will also help me in https://github.com/kata-containers/kata-containers/pull/7621 if you assume that it is not always a single-node cluster. However I can change it on my side if this PR gets merged first.